manager/manager.go

```diff
@@ -768,11 +783,44 @@ func (m *manager) registerCollectors(collectorConfigs map[string]string, cont *c
 	return nil
 }
 
+// Enables overwriting an existing containerData/Handler object for a given containerName.
+// can't use createContainer as it just returns if a given containerName has a handler already
+// Ex: rkt handler will want to take priority over the raw handler, but the raw handler might be created first
+
+// Only allow raw handler to be overriden
+func (m *manager) overrideContainer(containerName string, watchSource watcher.ContainerWatchSource) error {
```

Comment:
I thought I said this elsewhere, but can't find the comment.... Can we remove overrrideContainer for now (since it's unused), so that I can review it when I can see how it's actually used?